libraries/net/include/psibase/blocknet.hpp

```diff
@@ -202,6 +202,9 @@ namespace psibase::net
             }
          }
          assert(!"Unknown peer connection");
+
+         // Silence warning
+         return *std::make_unique<peer_connection>(0);
```

Comment:
Adding nonsense code is a poor way to remove warnings. Try `__builtin_unreachable`

Reply:
Ah, nice. Fixed